reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
As I understand `fields` is not defined in X3D, but `parts` is. So in this case renaming `fields` to `uniforms` would not break X3D compatibility, but only X3DOM one, right? Again, I believe it is simpler if we can be as close as possible to X3DOM...